### DISCUSSION TITLE
packages.py from wgrib2

### DISCUSSION THREAD

**Discussion by aveleda** [spack/spack#32856] [General]
Hi,

How to use the wgrib2 from NOAA-EMC  (https://github.com/NOAA-EMC/spack/blob/jcsda_emc_spack_stack/var/spack/repos/builtin/packages/wgrib2/package.py) instead default package.

Regards,

**Comment by climbfuji** [comment]
You can do something like this:
```
# in your spack clone
git remote add jcsda-emc https://github.com/noaa-emc/spack
git remote update
git checkout jcsda-emc/jcsda_emc_spack_stack -- var/spack/repos/builtin/packages/wgrib2/package.py
```
However, it would be much better if we (JCSDA/EMC) updated the wgrib2 package in the authoritative spack repo from our fork. Let me try to do that immediately.

**Reply by climbfuji** [reply]
See here https://github.com/spack/spack/pull/32857

**Reply by aveleda** [reply]
Thank you. 
It lacked to add the dependencies.

Regards,
